prusa/link/web/connection.py

```diff
@@ -122,9 +122,8 @@ def api_connection_set(req):
 
     app.settings.update_sections()
 
-    # Use values from settings file, strip off the quotes
-    name = printer_settings.name.strip('\"')
-    location = printer_settings.location.strip('\"')
+    name = printer_settings.name
+    location = printer_settings.location
```

Comment:
what about use `printer_settings.name` and `printer_settings.location` in calling compose_register_url?